apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifi_manager_main.c

```diff
@@ -389,11 +676,9 @@ static void itc_wifi_manager_reconnect_p(void)
 	TC_ASSERT_EQ_CLEANUP("wifi_manager_connect_ap", ret, WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
 	WIFITEST_WAIT;
 
-	sleep(SEC_5);
 	ret = wifi_manager_connect_ap(&config);
 	TC_ASSERT_EQ_CLEANUP("wifi_manager_connect_ap", ret, WIFI_MANAGER_SUCCESS, wifi_manager_disconnect_ap(); wifi_manager_deinit());
```

Comment:
This assert should be changed to TC_ASSERT_NEQ_CLEANUP, because a connection trial in the connected state returns WIFI_MANAGER_FAIL.